### PR TITLE
chore(docs)Adding pointers in Get started section

### PR DESCRIPTION
**Pull request by amyshwang**
1. Move Design for performance to Operations, having checked section description in introduction
2. Add a new section in the last page of Get started pointing to some concepts and pages

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -61,6 +60,8 @@ programming languages through our various network endpoints.
 This section contains resources for managing QuestDB instances and has dedicated
 pages for the following topics:
 
+- - [Design for performance](/docs/operations/design-for-performance) for configuring
```
Comment (puzpuzpuz) [inline docs/introduction.md:63]:
```suggestion
- [Design for performance](/docs/operations/design-for-performance) for configuring
```

**Review by amunra — CHANGES_REQUESTED** [review]
My suggestions are opinionated: Feel free to accept them / discard them as you see fit.

File: docs/get-started/learn-more.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+our [Capacity planning](/docs/operations/capacity-planning/) provides a more
+advanced optimization guide for using QuestDB in production environment.
+
+The below are a few concepts or SQL keywords that QuestDB creates for optimal
+time series performance:
+
+- [Symbol](/docs/concept/symbol/): a data type designed for repetitive strings
+  in high cardinality datasets
+- [Commit lag](/docs/guides/out-of-order-commit-lag/): a guide on how ILP
+  ingestion processes out-of-order data
```
[the remark this answers is not in the document]
Comment (amyshwang) [reply docs/get-started/learn-more.md:20]: I tweaked it a bit - 
"out-of-order data ingestion and associated configuration parameters."

File: docs/get-started/learn-more.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+To learn more about QuestDB and to create a proof of concept, please refer to
+[Design for performance](/docs/operations/design-for-performance/). In addition,
+our [Capacity planning](/docs/operations/capacity-planning/) provides a more
+advanced optimization guide for using QuestDB in production environment.
+
+The below are a few concepts or SQL keywords that QuestDB creates for optimal
+time series performance:
+
+- [Symbol](/docs/concept/symbol/): a data type designed for repetitive strings
```
[the remark this answers is not in the document]
Comment (amyshwang) [reply docs/get-started/learn-more.md]: Capitalised the first letters but I don't think full stops should be used as these are not complete sentences.

**Review by amunra — APPROVED** [review]
Lookin' good!

**Pull request merged**